source/optional_features/welsh_language/index.html.md.erb

```diff
@@ -22,4 +22,6 @@ When your user makes a payment, they will also see a
 payment pages, you may also want to use Welsh text in the
 `description`.
 
-GOV.UK Pay does not automatically translate emails into Welsh. Please [contact us](/support_contact_and_more_information/#contact-us) if you would like to send Welsh language emails to your users.
+GOV.UK Pay does not automatically translate emails into Welsh. If you want to send your own emails in Welsh, disable GOV.UK Pay sending emails by signing in to the [GOV.UK admin tool](https://selfservice.payments.service.gov.uk/login).
```

Comment:
it's more than signing in, is it 'by managing your settings in the admin tool' or something like that? But happy with what you recommend.

Reply:
The 'do xxx by signing into the admin tool' is our usual approach. But just to make it super-clear, in https://github.com/alphagov/pay-tech-docs/pull/367/commits/cb2ae90006810ea8ae4a2f96bd35d8bb1c4c792b, I've tweaked the link in so it goes straight to the email notifications admin page.

(I also thought about reversing the sentence to '..., sign into the admin tool to [turn off emails]', but I think that will make it sound too much like you can actually make Welsh emails happen somehow in the admin tool.)